test/test_torch.py

```diff
@@ -9040,6 +9040,17 @@ def test_kthvalue(self, device, dtype):
             self.assertEqual(res1val[:, :], res2val[:, :, k - 1], atol=0, rtol=0)
             self.assertEqual(res1ind[:, :], res2ind[:, :, k - 1], atol=0, rtol=0)
 
+    # test overlapping output
+    @dtypes(torch.double)
+    def test_kthvalue_overlap(self, device, dtype):
```

Comment:
I think this needs the `@onlyOnCPUAndCUDA` decorator, to avoid running on the XLA CI job, since this test is failing there.

Reply:
Done！